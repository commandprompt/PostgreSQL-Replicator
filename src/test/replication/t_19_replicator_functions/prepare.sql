CREATE LANGUAGE plpgsql;
CREATE SCHEMA "command prompt";
CREATE SCHEMA "123' junk";

CREATE TABLE "command prompt"."public projects"(id integer primary key, name varchar, active bool);
CREATE TABLE "command prompt"."secret projects"(id integer, name varchar, active bool);
CREATE TABLE "123' junk"."root passwords"(id integer primary key, pass varchar, fqdn varchar);

BEGIN;
ALTER TABLE "command prompt"."public projects" ENABLE REPLICATION;
ALTER TABLE "command prompt"."public projects" ENABLE REPLICATION ON SLAVE 0;
ALTER TABLE "command prompt"."public projects" ENABLE REPLICATION ON SLAVE 1;
ALTER TABLE "command prompt"."public projects" ENABLE REPLICATION ON SLAVE 2;
ALTER TABLE "command prompt"."public projects" ENABLE REPLICATION ON SLAVE 3;

ALTER TABLE "123' junk"."root passwords" ENABLE REPLICATION;
ALTER TABLE "123' junk"."root passwords" ENABLE REPLICATION ON SLAVE 0;
ALTER TABLE "123' junk"."root passwords" ENABLE REPLICATION ON SLAVE 1;
ALTER TABLE "123' junk"."root passwords" ENABLE REPLICATION ON SLAVE 2;
ALTER TABLE "123' junk"."root passwords" ENABLE REPLICATION ON SLAVE 3;
COMMIT;

