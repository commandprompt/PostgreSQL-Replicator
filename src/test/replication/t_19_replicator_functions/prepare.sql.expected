psql:.*:5: NOTICE:  CREATE TABLE / PRIMARY KEY will create implicit index "public projects_pkey" for table "public projects"
psql:.*:7: NOTICE:  CREATE TABLE / PRIMARY KEY will create implicit index "root passwords_pkey" for table "root passwords"
